build/yamls/flow-aggregator/patches/release/.gitignore

```diff
@@ -0,0 +1 @@
+# placeholder
```

Comment:
I believe the convention is an empty file called `.gitkeep`

Reply:
ok, I was following the style in build/yamls/octant/ or windows/, could you please check again?

Reply:
I added the .gitignore file before I became aware of the .gitkeep "convention", but that's a pretty minor point anyway. Feel free to update all of them to .gitkeep as part of this PR or keep is as it is.